cmd/openshift-install/gather.go

```diff
@@ -95,30 +96,30 @@ func runGatherBootstrapCmd(directory string) error {
 	if err != nil {
 		if err2, ok := err.(errUnSupportedGatherPlatform); ok {
 			logrus.Error(err2)
-			return unSupportedPlatformGather()
+			return unSupportedPlatformGather(directory)
 		}
 		return errors.Wrapf(err, "failed to get bootstrap and control plane host addresses from %q", tfStateFilePath)
 	}
 
-	logGatherBootstrap(bootstrap, port, masters)
+	logGatherBootstrap(bootstrap, port, masters, directory)
 	return nil
 }
 
-func logGatherBootstrap(bootstrap string, port int, masters []string) {
-	var (
-		sshport string
-		scpport string
-	)
-	if port != 22 {
-		sshport = fmt.Sprintf(" -p %d", port)
-		scpport = fmt.Sprintf(" -P %d", port)
+func logGatherBootstrap(bootstrap string, port int, masters []string, directory string) {
+	logrus.Info("Pulling logs from bootstrap for debugging")
+
+	client, err := ssh.NewClient("core", fmt.Sprintf("%s:%d", bootstrap, port), nil)
```

Comment:
I'm fine with this for a first iteration, but in the future I'd like a way to load only the private key that matches the public key configured for the cluster.  Not sure if we want to add at least some of that check now so we can error gracefully in the "you didn't configure a SSH key when you launched this cluster, so sorry, no logs for you" case.

Reply:
https://github.com/openshift/installer/pull/1822/commits/16c8fee3bd84fb272d50a107abc544c624bc26d6 allows you to use a specific key if you like, otherwise we can be optimistic and try with all.